frontend/packages/client/public/manifest.json

```diff
@@ -11,5 +12,13 @@
   "start_url": ".",
   "display": "standalone",
   "theme_color": "#000000",
-  "background_color": "#ffffff"
+  "background_color": "#ffffff",
+  "permissions": ["storage", "activeTab", "alarms"],
+  "web_accessible_resources": [
+    {
+      "resources": ["*.html"],
+      "matches": ["<all_urls>"],
+      "extension_ids": ["hpclkefagolihohboafpheddmmgdffjm"]
+    }
+  ]
```

Comment:
there seems to be an issue when trying to connect with Lilico on testnet and loading the chrome extesion:
```
Denying load of chrome-extension://hpclkefagolihohboafpheddmmgdffjm/popup.html?l6n=http%3A%2F%2Flocalhost%3A3000. Resources must be listed in the web_accessible_resources manifest key in order to be loaded by pages outside the extension.
```
I've tried the configuration  above with no success, I still get the error

Reply:
Bumping to FCL `v1.2.0` resolves. No need to update the Cast manifest